### PR TITLE
Added several kernels

### PR DESCRIPTION
**Pull request by abotiamnot**
Added,

- Clementine

- Deep Impact( Load them in the following order, dii_preenc174_nav_v1.bsp, dif_preenc174_nav_v1.bsp, di_tempel1_ssd_v1.bsp, di_finalenc_nav_v3_to06048.bsp for maximum accuracy)

- Deep Space

- EPOXI

- Hayabusa ( Load them in the following order, de403s.bsp, sb_25143_140.bsp, hay_jaxa_050916_051119_v1n.bsp, hay_osbj_050911_051118_v1n.bsp for the data )

EDIT : The list looks ugly, need to improve it.

### CODE REVIEW

**Comment by abotiamnot** [comment]
Hmm, something wrong with the docstring I guess

**Comment by dstansby** [comment]
I think the problem is when there are multiple kernels for one spacecraft. I'll try and find a fix and let you know.

**Comment by dstansby** [comment]
Okay, once #390 is merged having multiple kernels under one name should be fine.

**Comment by abotiamnot** [comment]
Thanks for mixing that error!

File: heliopy/data/spice.py [hunk 1]
```diff
@@ -30,7 +30,23 @@
                      'Helios 1_Predicted': ['https://naif.jpl.nasa.gov/pub/naif/HELIOS/kernels/spk/160707AP_helios1_81272_86074.bsp'],
                      'Helios 2': ['https://naif.jpl.nasa.gov/pub/naif/HELIOS/kernels/spk/100607R_helios2_76016_80068.bsp'],
                      'Juno_Reconstructed': ['https://naif.jpl.nasa.gov/pub/naif/JUNO/kernels/spk/juno_rec_orbit.bsp'],
-                     'Juno_Predicted': ['https://naif.jpl.nasa.gov/pub/naif/JUNO/kernels/spk/juno_pred_orbit.bsp']}
+                     'Juno_Predicted': ['https://naif.jpl.nasa.gov/pub/naif/JUNO/kernels/spk/juno_pred_orbit.bsp'],
+                     'Clementine_nrl': ['https://naif.jpl.nasa.gov/pub/naif/pds/data/clem1-l-spice-6-v1.0/clsp_1000/data/spk/clem_jpl.bsp'],
```
Comment (dstansby) [inline heliopy/data/spice.py]: Should this be `jpl` instead of `nrl`? (that's what's in the URL)

File: heliopy/data/spice.py [hunk 2]
```diff
@@ -30,7 +30,23 @@
...
+                     'Clementine_gsfc': ['https://naif.jpl.nasa.gov/pub/naif/pds/data/clem1-l-spice-6-v1.0/clsp_1000/data/spk/clem_gsfc.bsp'],
+                     'Clementine_jpl': ['https://naif.jpl.nasa.gov/pub/naif/pds/data/clem1-l-spice-6-v1.0/clsp_1000/data/spk/clem_jpl.bsp'],
+                     'de414': ['https://naif.jpl.nasa.gov/pub/naif/pds/data/clem1-l-spice-6-v1.0/clsp_1000/data/spk/de414.bsp'],
```
Comment (dstansby) [inline heliopy/data/spice.py]: I think all the de* kernels shouldn't be added, since they're not spacecraft trajectories (as far as I know)

Comment (abotiamnot) [reply heliopy/data/spice.py]: Ah yes, they're to be loaded along with another kernel, I've added those multiple links in the required kernels so I will remove them

**Comment by abotiamnot** [comment]
@dstansby  Modified the CSV table a bit, can you take a look?

**Comment by abotiamnot** [comment]
@dstansby Can you take a look at this? I've modified the table a bit.

**Comment by abotiamnot** [comment]
#387

**Comment by dstansby** [comment]
Hmm, I think the table looks a bit odd: https://114-122063028-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/data/spice.html

Maybe there should be separate columns for names and identifiers, and the URLs should just be some short text with a link (maybe numbers like 1, 2).

**Comment by abotiamnot** [comment]
I can't seem to figure out how to add links to the table using _doc_

**Comment by abotiamnot** [comment]
Done! But I wonder how to add hyperlinks to the table using the __doc__

**Pull request merged**